### ISSUE TITLE
Some question about Finetuning with New Dataset

### ISSUE THREAD

**Issue by shiqitannn** [thu-ml/RoboticsDiffusionTransformer#37]
Thank you for sharing the  great research. I have been working on fine-tuning the 1B model with a new dataset,  my setting is:
- number of train episodes: 367
- number of validation episodes: 41
- a single task with a fixed instruction
- batch size 8,  train on 8 A800,  so the total batch size is 64
- train steps: 10w 
I found that the model fits very well on the training set, but performs poorly on the validation set and poorly when deploy in real robots.
![train](https://github.com/user-attachments/assets/cdf7260d-f5ab-46b1-8686-05f1f392bfbb)
![val](https://github.com/user-attachments/assets/2945b3af-f987-42db-b68a-cd0b5d72b499)

 I want to know where my problem lies and whether the fine-tuning process is reasonable. I look forward to your reply.

**Comment by shiqitannn** [comment]
And I would like to ask a question regarding this issue: On the A800, when I increased the batch size from 8 to 32, the training time also increased fourfold. Is this normal?

**Comment by lanlankilkil** [comment]
I'd like to ask what type of robotic arm you trained with, and is it necessary to collect so many episodes for a single task，and ow long have you been training at 8-a800?

**Comment by csuastt** [comment]
Have you considered using a much smaller model, like RDT-170m?

Btw, we will release our simulation benchmark within a week. You may have a try as a reference.

**Comment by ethan-iai** [comment]
After private discussion, this issue has been resolved.

**Comment by lanlankilkil** [comment]
Hello, I would like to inquire about some questions regarding RDT. Could we exchange contact information?

**Comment by csuastt** [comment]
@lanlankilkil https://rdt-robotics.github.io/rdt-robotics/wechat.html

**Comment by chenyt31** [comment]
The WeChat group QR code has expired. Would you mind sending the group again? @csuastt

**Comment by csuastt** [comment]
@chenyt31 

![Image](https://github.com/user-attachments/assets/f6915a0a-f0b1-4252-a66a-0a25c65a716f)
